high_card.rb

```diff
@@ -1,5 +1,21 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+
+
+ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K"]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
-players = []
+deck = []
+deck = ranks.product(suits)
```

Comment:
This works, and is absolutely the best way. But to get practice, how would you write this using just the `.each` method?